cmd/minikube/cmd/config/config_test.go

```diff
@@ -41,12 +41,14 @@ var configTestCases = []configTestCase{
     "ReminderWaitPeriodInHours": 99,
     "cpus": 4,
     "disk-size": "20g",
+    "iso-url": "http://foo.bar/minishift-centos.iso",
     "log_dir": "/etc/hosts",
     "show-libmachine-logs": true,
     "v": 5,
     "vm-driver": "kvm"
 }`,
 		config: map[string]interface{}{
+			"iso-url": "http://foo.bar/minishift-centos.iso",
```

Comment:
I'd like to see some tests also persisting actually to disk where on then verifies the content via reading from disk (potentially utilizing _MINISHIFT_HOME_ to make sure the config file gets somewhere into a known location. Maybe not a must have right now, but it would be nice. Right now we just do everything in memory.